### ISSUE TITLE
Error on http://www.the-button.com/

### ISSUE THREAD

**Issue by 2m** [ruffle-rs/ruffle#6679]
### Describe the bug

The following errors are reported to console when loading http://www.the-button.com:

```
AVM2 error: Property does not exist: Multiname { ns: [Package("flash.net")], name: Some("URLLoader"), params: [] } [966.js:1:27086](moz-extension://e7d44c36-c439-4448-aed7-18a7b2d8aa1d/dist/966.js)
AVM2 error: Property does not exist: Multiname { ns: [Package("flash.net")], name: Some("URLLoader"), params: [] } [966.js:1:27086](moz-extension://e7d44c36-c439-4448-aed7-18a7b2d8aa1d/dist/966.js)
AVM2 error: Property does not exist: Multiname { ns: [Package("flash.net")], name: Some("URLLoader"), params: [] } [966.js:1:27086](moz-extension://e7d44c36-c439-4448-aed7-18a7b2d8aa1d/dist/966.js)
Error occured when running AVM2 frame script: Property does not exist: Multiname { ns: [Package("flash.net")], name: Some("URLLoader"), params: [] } [966.js:1:27086](moz-extension://e7d44c36-c439-4448-aed7-18a7b2d8aa1d/dist/966.js)
```

### Expected behavior

Hatch simulator runs and allows to enter the numbers.

### Affected platform

Browser's extension

### Operating system

macOS

### Browser

Firefox 98

### Additional information

Using latest nightly web extension.

**Comment by Lord-McSweeney** [comment]
The SWF now hangs at "Initializing...". This may not be Ruffle's fault (some API within the server is returning responses that look broken).

**Comment by 2m** [comment]
That's the actual response that the client is expecting to get. :) The API was designed in 2007 and JSON was not that popular back then.

What would be the best way to debug this and see at what point it stops? I see that it is possible to run native app in verbose mode. Is something similar possible when running as extension?

**Comment by Lord-McSweeney** [comment]
Unfortunately, Ruffle doesn't offer very many debugger options. Verbose mode for the browser extension can be turned on by setting the log level to info in Ruffle's settings page.

One way to figure out where the SWF stops would be to set log level to info, then edit the SWF (using FFDEC) with trace statements to check if Ruffle can run that part of the code.

**Comment by 2m** [comment]
Last week we figured out that it was the missing support of `URLLoaderDataFormat.VARIABLES` that was the last piece of the puzzle to make this SWF run. I have added the support [here](https://github.com/2m/ruffle/commit/0c95f67a91c938daf753efe447dfd432615ba300) and deployed that to the website.

This change is still only in a branch of my fork. We [discussed it in Discord](https://discord.com/channels/610531541889581066/1018988594259574845/1078361441087803432) that it would be better to refactor it to UrlVariables before submitting PR. I will try to work on this in the following weekend, unless some else beats me to it. ;)
